lib/renderer.js

```diff
@@ -55,7 +68,7 @@ const renderRecipes = (recipes) => {
 }
 
 const renderGroceryList = (groceries) => {
-  JSON.parse(groceries).all_Groceries.map(grocery => {
+  JSON.parse(groceries).grocery_list.map(grocery => {
```

Comment:
I'm hoping this worked on your end. I had some trouble last night getting this set back up again after merging a branch I was working on with master. I think the issue now is getting the list to render on list window load. I'm not sure how important that is though.